examples/mp4.rs

```diff
@@ -76,21 +79,30 @@ fn start_example_client_code() -> Result<()> {
             },
             "encoder_preset": "medium",
             "initial": {
-                "type": "shader",
-                "id": "shader_node_1",
-                "shader_id": "shader_example_1",
-                "children": [
-                    {
-                        "id": "input_1",
-                        "type": "input_stream",
-                        "input_id": "input_1",
-                    }
-                ],
-                "resolution": { "width": VIDEO_RESOLUTION.width, "height": VIDEO_RESOLUTION.height },
+                "id": "input_1",
+                "type": "input_stream",
+                "input_id": "input_1",
             }
         }
     }))?;
 
+    info!("[example] Send register output audio request.");
+    common::post(&json!({
+        "type": "register",
+        "entity_type": "output_stream",
+        "output_id": "output_2",
+        "port": 8004,
+        "ip": "127.0.0.1",
+        "audio": {
+            "initial": {
+                "inputs": [
+                    {"input_id": "input_1"}
+                ]
+            },
+            "channels": "stereo"
+        }
+    }))?;
```

Comment:
1. It's easier to test synchronization without this weird shader.
2. I'll change examples soon anyway